buildlib/pr/tests.yml

```diff
@@ -28,6 +28,7 @@ jobs:
       - checkout: self
         clean: true
         fetchDepth: 100
```

Comment:
shall we also reduce `fetchDepth` to help in saving bandwidth? why not using `fetchDepth: 1`?

Reply:
if we don't need tags maybe `fetchTags: false`